test/sample-test.ts

```diff
@@ -1,10 +1,12 @@
-import { expect } from 'chai';
-import { ethers } from 'hardhat';
+import { expect } from "chai";
+import { ethers } from "hardhat";
```

Comment:
i think single quote should be better

Reply:
It was a formatting omission. My apologies!